packages/components/upload/__tests__/upload-dragger.spec.ts

```diff
@@ -9,7 +10,7 @@ const Wrapper = defineComponent({
     onDrop: Function,
   },
   setup(props, { slots }) {
-    provide('uploader', { accept: 'video/*' })
+    provide(uploadContextKey, { accept: computed(() => 'video/*') })
     return () => h(UploadDragger, props, slots)
```

Comment:
Since we are refactoring this component, do you think we can use `JSX` for this file?

Reply:
It's not necessary, and using JSX would add even more complexity.